x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java

```diff
@@ -77,13 +78,17 @@
 import static org.elasticsearch.xpack.application.connector.ConnectorFiltering.fromXContentBytesConnectorFiltering;
 import static org.elasticsearch.xpack.application.connector.ConnectorFiltering.sortFilteringRulesByOrder;
 import static org.elasticsearch.xpack.application.connector.ConnectorTemplateRegistry.MANAGED_CONNECTOR_INDEX_PREFIX;
+import static org.elasticsearch.xpack.core.ClientHelper.CONNECTORS_ORIGIN;
 
 /**
  * A service that manages persistent {@link Connector} configurations.
  */
 public class ConnectorIndexService {
 
+    // The client to perform any operations on user indices (alias, ...).
     private final Client client;
```

Comment:
It looks like the only usage of this is `new ConnectorSyncJobIndexService(client)`. But could that just as easily take `clientWithOrigin`? And if so, you don't need two `Client` instance variables?